ComponentKit/Debug/CKComponentDebugController.mm

```diff
@@ -109,9 +111,6 @@ + (BOOL)debugMode
     return context; // no need for a debug view if the component has a view.
   }
 
-  static CK::StaticMutex l = CK_MUTEX_INITIALIZER;
-  CK::StaticMutexLocker lock(l);
```

Comment:
Do we want to assert this is called on the main thread? Or do you think that's overkill in this case?